internal/app/backend/backend_service.go

```diff
@@ -125,10 +134,15 @@ func (s *backendService) FetchMatches(ctx context.Context, req *pb.FetchMatchesR
 		results = resp.Match
 	}
 
+	err = doFetchMatchesAddSkiplistIds(ctx, s.store, results)
```

Comment:
This is the only task to be done in backend. Btw, why have this helper to 'doFetchMatchesAddSkiplistIds' instead of simply having a loop here that adds tickets to ignored list directly?

Reply:
To make the code more testable. We have an evaluator invocation before this helper call so unit test cant cover this line without hiding the context inside a function.